cmd/locket/main.go

```diff
@@ -179,8 +179,12 @@ func main() {
 		os.Exit(1)
```

Comment:
Are these `os.Exit`s superfluous?

Reply:
They are, since `logger.Fatal()` will do it once sent. I think some tooling prompted earlier Terin into adding them to make the flow control clearer, but nothing was yelling at me at the new one I added elsewhere.